lib/utils/axios-helpers.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios');
```

Comment:
Created a separate file as I was getting weird errors in certain tests where a function was undefined.